include/swift/AST/ExtInfo.h

```diff
@@ -1140,6 +1140,14 @@ class SILExtInfoBuilder {
                                     : (bits & ~ErasedIsolationMask),
                              clangTypeInfo, lifetimeDependenceInfo);
   }
+
+  [[nodiscard]]
+  SILExtInfoBuilder withSendable(bool sendable = true) const {
+    return SILExtInfoBuilder(sendable ? (bits | SendableMask)
+                                      : (bits & ~SendableMask),
+                             clangTypeInfo, lifetimeDependenceInfo);
+  }
+
```

Comment:
Isn't this doing the same thing as  `SILExtInfoBuilder withConcurrent(bool isSendable = true) const`?

Reply:
Should we just rename `withConcurrent` to be `withSendable`?

Reply:
Ah, I didn't see that.

> Should we just rename withConcurrent to be withSendable?

yes. Done

Reply:
thanks! lgtm :)